src/selectors.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  getRoute: (state) => state.router.route,
+  getRoutes: (state) => state.router.routes,
+  getRouteName: (state) => state.router.route.name,
+  getRoutePath: (state) => state.router.route.path,
+  getQuery: (state) => state.router.route.query,
+  getQueryParam: (state, name) => state.router.route.query[name],
```

Comment:
Should return `null` if the param name doesn't exist.
Also, all those should be unit tested.